sdk/tables/Azure.Data.Tables/tests/TableServiceLiveTestsBase.cs

```diff
@@ -61,6 +61,8 @@ public TableServiceLiveTestsBase(bool isAsync, TableEndpointType endpointType) :
             {"GetPropertiesReturnsProperties", "GetProperties is currently not supported by Cosmos endpoints."},
             {"GetTableServiceStatsReturnsStats", "GetStatistics is currently not supported by Cosmos endpoints."},
             {"ValidateSasCredentialsWithRowKeyAndPartitionKeyRanges", "Shared access signature with PartitionKey or RowKey are not supported"},
+            {"ValidateAccountSasCredentialsWithPermissions", "SaS for account operations not supported"},
```

Comment:
The nittiest of nits:  I would think this is "SAS" here... why the lowercase "a" ?